components/Questionnaire/Confirmation/index.vue

```diff
@@ -14,188 +14,189 @@
-                }"
-              >
-                <object
-                  v-if="file.source"
-                  :data="file.source"
-                  class="registration__form__image--attached-uploaded"
-                  width="88"
-                  height="88"
-                />
-                <img
-                  v-else
-                  height="22"
-                  width="22"
-                  src="@/assets/icons/IconPdf.svg"
-                  alt="Surat Keterangan"
-                >
-              </div>
+          <div class="registration__form-content--container">
+            <div class="registration__form-content--container-title">
+              Informasi Umum
             </div>
-            <div class="registration__form-col-desc">
-              <div class="registration__form__subtitle">
-                Unggah SK Pengangkatan Kepala Desa
-              </div>
-              <div class="registration__form__placeholder">
-                File yang didukung adalah .pdf maksimal 10 Mb.
-              </div>
-              <div class="registration__form__button grid grid-cols-5">
-                <div class="col-span-1">
-                  <button class="registration__form__button-btn text-sm" type="button" @click="$refs.letter.click()">
-                    Unggah Surat
-                    <jds-icon class="ml-1" size="10px" name="plus-bold" />
-                  </button>
-                  <input
-                    ref="letter"
-                    type="file"
-                    hidden="true"
-                    accept="application/pdf"
-                    @change="onFileChange()"
+            <BaseInput
+              v-model="params.nama"
+              class="mt-2"
+              label="Nama Lengkap"
+              type="text"
+              autofocus
+              autocomplete="name"
+              placeholder="Cth: Agus Permadi"
+              :error="!!(errors.name)"
+              :error-message="errors.name"
+            />
+            <BaseInput
+              v-model="params.posisi"
+              class="mt-2"
+              label="Jabatan"
+              type="text"
+              autofocus
+              autocomplete="jabatan"
+              placeholder="Cth: Kepala Desa Manyeti"
+              :error="!!(errors.position)"
+              :error-message="errors.position"
+            />
+            <div class="grid grid-cols-5 mt-6">
+              <div class="registration__form-col-image">
+                <div
+                  :class="{
+                    'registration__form__image': true,
+                    'registration__form__image--attached': file.isAttached
+                  }"
+                >
+                  <object
+                    v-if="file.source"
+                    :data="file.source"
+                    class="registration__form__image--attached-uploaded"
+                    width="88"
+                    height="88"
+                  />
```

Comment:
cool, I never use this `object` element before ❤️

Reply:
I didn't even know it existed 😳